tests/plugins/testplugin.py

```diff
@@ -37,6 +37,12 @@ def get_category(self):
     def _get_streams(self):
         if "empty" in self.url:
             return
+        if "UnsortableStreamNames" in self.url:
+            return {
+                "vod": HTTPStream(self.session, "http://test.se/stream"),
+                "vod_alt": HTTPStream(self.session, "http://test.se/stream"),
```

Comment:
the `_alt` tag is to early

you are here

https://github.com/streamlink/streamlink/blob/cd6f94a5a4ee83cb5b61a295df9e13956bcfcb27/src/streamlink/plugin/plugin.py#L317

and they are all called `vod` here

it will be added later

https://github.com/streamlink/streamlink/blob/cd6f94a5a4ee83cb5b61a295df9e13956bcfcb27/src/streamlink/plugin/plugin.py#L358-L366

---

you might use a generator instead of a dict here

```py
        if "UnsortableStreamNames" in self.url:
            def gen():
                for i in range(3):
                    yield "vod", HTTPStream(self.session, "http://test.se/stream")
            return gen()
```